tests/fast-integration/utils/index.js

```diff
@@ -315,6 +315,11 @@ async function k8sApply(resources, namespace, patch = true) {
 }
 
 function waitForK8sObject(path, query, checkFn, timeout, timeoutMsg) {
+  return waitForK8sObjectWithWatch(path, query, checkFn, timeout, timeoutMsg, watch);
+}
+
+// Allows to pass watch with different than global K8S context.
+function waitForK8sObjectWithWatch(path, query, checkFn, timeout, timeoutMsg, watch) {
```

Comment:
Remove the function. Use the default parameter instead.

Reply:
Corrected. Thank you!